### ISSUE TITLE
Option to zoom

### ISSUE THREAD

**Issue by Jihefel** [flip-forge/vue-flip-forge#1]
Do you plan to integrate a zoom function on the pages (e.g. 2 zoom levels and then a complete zoom out)? 
That would be great.
Thank you for the work

**Comment by alexkiro** [comment]
Yes definetly, a zoom function is something that I plan to integrate!
